index.css

```diff
@@ -19,14 +19,12 @@ body {
 }
 
```

Comment:
General note,  appears the burger icon doesn't expand on it's own when clicked without "js assist."   That is probably in bootstrap.js.  But I bet we can write some custom js to pop it open.  I'll investigate later.